### DISCUSSION TITLE
Prompt in loop

### DISCUSSION THREAD

**Discussion by amangit007** [microsoft/BotFramework-Composer#9709] [Q&A]
I am having a loop inside another loop in bot framework composer. The inner loop completes it's iteration if the flow inside it only includes actions like set property or send messages. The moment I use ask question prompt (any question prompt normal or adaptive card) inside that inner loop the loop completes the first iteration and then breaks the loop. So is it a bug or I am doing something wrong.

**Comment by OEvgeny** [comment]
Have you tried the "Continue loop" node? See the documentation here: https://learn.microsoft.com/en-us/composer/how-to-control-conversation-flow?tabs=v2x#use-loop-actions
